src/main/resources/application.properties

```diff
@@ -1,7 +1,9 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/project_bweng
+spring.datasource.url=jdbc:mariadb://localhost:3306/project_bweng
 spring.datasource.username=user
 spring.datasource.password=pw
-#spring.datasource.driver-class-name=com.mysql.jdbc.Driver
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL57Dialect
-spring.jpa.hibernate.ddl-auto=create
-server.port=8081
\ No newline at end of file
+spring.datasource.driver-class-name=org.mariadb.jdbc.Driver
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MariaDBDialect
+spring.jpa.hibernate.ddl-auto=update
+server.port=8081
+
+security.jwt.secret=averysecretsecretnoonewilleverguess
```

Comment:
Maybe should be a placeholder instead of a real value?

Reply:
What do you mean by a placeholder in this case? As far as I know the value of this property is mandotary for the signature part of the JWT.